### DISCUSSION TITLE
I Created a Tool to Export Kutt Links as Snapp-Compatible CSV

### DISCUSSION THREAD

**Discussion by IvanLi-CN** [urania-dev/snapp#55] [Show and tell]
Hello, community! I've built a tool to migrate short links from Kutt to Snapp. It's available as a Node package.

[kutt-to-snapp - npm](https://www.npmjs.com/package/kutt-to-snapp)

Usage:

```shell
KUTT_URL=https://kutt.it \
KUTT_TOKEN=your-kutt-token \
MIGRATE_ALL=true \
npx kutt-to-snapp > kutt.csv
```

After running this, you just need to import the kutt.csv file into snapp to complete the migration.

I initially created this tool for my own use, but I thought it might be helpful for others, so I decided to publish it.

Feel free to check it out if you need to migrate your short links from Kutt to Snapp. I hope some of you find it useful!

**Comment by urania-dev** [comment]
Thank you very much!

**Comment by urania-dev** [comment]
Based on your comment idea to be able to migrate across different platform I tried to recreate a ui importer that could be more elastic in 0-9rc
